### ISSUE TITLE
Overriding AuthorizationsController in Development Mode

### ISSUE THREAD

**Issue by zacheryph** [doorkeeper-gem/doorkeeper-openid_connect#158]
We are experiencing an issue in development mode where AuthorizationsController is losing touch with its ancestory. It seems to be due this gem overriding the class directly instead of mixing into and/or not defining a super class. making any changes in development requires reloading the entire application. You can replicate this by doing `rails new`, adding doorkeeper and this gem, doing `doorkeeper:install` and then doing the below in console.

This causes issues with NoMethodError `binary_params_for?` not being defined during requests.

```ruby
Doorkeeper::AuthorizationsController.superclass
#=> Doorkeeper::ApplicationController
reload!
# Reloading...
#=> true
Doorkeeper::AuthorizationsController.superclass
#=> Object
```

**Comment by nbulaj** [comment]
Hi @zacheryph . https://github.com/doorkeeper-gem/doorkeeper-openid_connect/pull/173 should fix the issue so you can try it from the branch (for now) or with a release later and report any issues found. Thanks!

**Comment by nbulaj** [comment]
Released in 1.8.2
